docker-compose/mainnet/docker-compose.yml

```diff
@@ -89,6 +75,16 @@ services:
       - SECRET_SRC
       - KEYSTORE_FILE
       - KEYSTORE_PWD
+      - BOOTSTRAP_TIMEOUT
+      - WATCHER_PREPARATION_TIMEOUT
+      - WATCHER_VALIDATION_TIMEOUT
+      - DATABASE_INTERACTION_TIMEOUT
+      - MINING_WAIT_TIMEOUT
+      - DATABASE_CONNECTION_TIMEOUT
+      - SERVER_READ_HEADER_TIMEOUT
```

Comment:
As those timeouts coming from env vars we can have validator for the timeouts vars, to make sure that the set timeout is inside a valide range.
Avoiding for exemple in the PEGOUT_DEPOSIT_CHECK_TIMEOUT user put a high value that can make application not behave in the expected way.